com.unity.render-pipelines.universal/Shaders/ShadowCasterPass.hlsl

```diff
@@ -27,10 +27,12 @@ float4 GetShadowPositionHClip(Attributes input)
 
     float4 positionCS = TransformWorldToHClip(ApplyShadowBias(positionWS, normalWS, _LightDirection));
 
+#if defined(_SHADOW_PANCAKING_ON)
```

Comment:
The way this is define might change the existing behaviour for custom user shaders. 
F.ex, a user has a custom shader that includes ShadowCaster pass. Their material doesn't set the _SHADOW_PANCAKING_ON keyword. Once they update to this URP version their custom shadow shaders will stop doing pancaking and that will be a regression. 

What we should do is to introduce a keyword to disable shadow_pancaking instead, that way the default behaviour when not defining the keyword is to do pancaking, keeping existing behaviour for custom shaders unchanged.

Reply:
Switching this to a disable definitely sounds like a good idea!

Reply:
That's a bit unfortunate. Having randomly positive/negative conditional defines will make a mess as a whole. Is there no other option to keep the condition positive? Can we upgrade materials to have it enabled by default?